.github/workflows/run-test-sh.yml

```diff
@@ -0,0 +1,39 @@
+name: run-test-shell
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable
+
+  push:
+    branches:
+      - master
+      - stable
+
+jobs:
+  main:
+    name: Build and test image
```

Comment:
I try to keep those names short otherwise you don't see the matrix args in the "Status check" summary.